spec/ivapi/error_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+describe Ivapi::Error do
+  describe '.from_response' do
+    it 'has bad request error raised when status is 400' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.